Automattic/woocommerce-payments
# feat: add onboarding UPE payment methods

Fixes https://github.com/Automattic/woocommerce-payments/issues/1735

#### Changes proposed in this Pull Request









Incremental step.
Fetches & persists the payment methods for UPE during the onboarding experience.

#### Testing instructions





- Ensure you have the latest version of the dev tools: `cd docker/wordpress/wp-content/plugins/woocommerce-payments-dev-tools && git checkout trunk && git pull`
- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Ensure you have the "Add UPE additional payment methods" flag enabled in WCPay Dev Tools
- Visit http://localhost:8082/wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods
- Enable some of the payment methods: https://d.pr/i/iJktEJ
- Click "Continue"
- Refresh the page
- The settings are persisted
- On the settings page, the same payment methods should be displayed
- Please note this is not yet working for the 1-click checkout method(s)

-------------------
